NMTest.DataSource/DatabaseStore.cs

```diff
@@ -1,19 +1,18 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿using System.Collections.Generic;
 using System.Threading;
 
 namespace NMTest.DataSource
 {
     public class DatabaseStore
     {
+        private readonly Dictionary<string, object> _values = new Dictionary<string, object>();

```

Comment:
I'm inclined to leave the `private` access modifiers absent, and instead leave those as bonus points for the candidate.